### ISSUE TITLE
Kodi Nexus and Flatpak Kodi crash a few seconds after start screen in Arch linux

### ISSUE THREAD

**Issue by cg00001** [xbmc/xbmc#22213]
### Describe the bug
Here is a clear and concise description of what the problem is:

This has happened again in the past, and it seems again something related to mesa causes the crashes.
It takes from 3 to 10+ tries to successfully open Kodi without crashes.
`2022-12-03 23:04:49.836 T:13484   error <general>: DBus error: org.freedesktop.DBus.Error.InvalidArgs - No such property “CanSuspend”  
2022-12-03 23:04:49.836 T:13484   error <general>: DBus error: org.freedesktop.DBus.Error.InvalidArgs - No such property “CanHibernate”`

### Your Environment
Used Operating system:

 - [ ] Android
 - [ ] iOS
 - [ ] tvOS
 - [x] Linux
 - [ ] OSX
 - [ ] Windows
 - [ ] Windows UWP

 - Operating system version/name: Arch linux
 - Kodi version: Kodi Nexus Beta 1, Flatpak Kodi

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by cg00001** [comment]
Just Kodi with a skin and a few addons from Kodi repository.

Crash log:
[kodi_crashlog-20221205_195152.log](https://github.com/xbmc/xbmc/files/10156630/kodi_crashlog-20221205_195152.log)

Kodi log:
[kodi.log](https://github.com/xbmc/xbmc/files/10156631/kodi.log)

**Comment by cg00001** [comment]
@a1rwulf, I figured it out myself.
I was using the libva-intel-driver-g45-h264 that caused that, I replaced it with the standard libva-intel-driver and the issue was gone.

**Comment by DaVukovic** [comment]
`gdb not installed, can't get stack trace.` 

Could you please install `gdb` and get us a new crashlog? Crashlogs without `gdb` installed are useless, unfortunately.

**Comment by cg00001** [comment]
> `gdb not installed, can't get stack trace.`
> 
> Could you please install `gdb` and get us a new crashlog? Crashlogs without `gdb` installed are useless, unfortunately.

Sorry, this is Kodi Nexus Beta 1, with a skin and a few addons from Kodi repository:

Crash log:
[kodi_crashlog-20221206_233841.log](https://github.com/xbmc/xbmc/files/10170496/kodi_crashlog-20221206_233841.log)


Kodi log:
[kodi.log](https://github.com/xbmc/xbmc/files/10170499/kodi.log)

**Comment by DaVukovic** [comment]
I highly guess your issue is python related 

Please take a look at this thread: https://forum.kodi.tv/showthread.php?tid=368069

I guess your python version is something <3.10.5 as that version should include the fix.

**Comment by cg00001** [comment]
> I highly guess your issue is python related
> 
> Please take a look at this thread: https://forum.kodi.tv/showthread.php?tid=368069
> 
> I guess your python version is something <3.10.5 as that version should include the fix.

My python in Arch is the latest -I do daily updates. 3.10.8-3, but I haven't deleted the whole .kodi folder and try from scratch. Don't know if existing Kodi test folders are influenced by python changes -or GPU driver changes, I changed mine to a basic driver just in case.
I checked my Flatpak installed packages and found double freedesktop packages with different versions after a simple update, so I uninstalled all and made a new test Kodi that seems to not crash. Flatpak (stable) uses Matrix with python  v3.9.
Thank you.
